Remove unused usings

jasonmalinowski opened the pull request:
This unbreaks the master-vs-deps correctness leg.

mavasani commented:
I am going to change remove unused usings to be a non-build breaking rule and open a separate work item to implement a code cleanup bot for removing these as suggested by @tmat.

jasonmalinowski commented:
@mavasani Did the policy recently change to make this breaking? Still not entirely sure how this one got through CI.

JoeRobich commented:
Those usings were probably missing in master where the rule was enabled. I've actually removed them from master-vs-deps already as part of the master-to-master-vs-deps merge pr

jasonmalinowski commented:
Ah, so that code I think is only in master-vs-deps, and these are the same files I touched in 52de7806c01bb3fc03b89755bab9dc0da615d525. So I think here's what happened:

1. I moved some types to different namespaces in master.
2. That got incorrectly merged to master-vs-deps because the CI states got mixed up by GitHub and it got auto-merged.
3. In that commit, I added the missing using statements, but didn't remove the unnecessary ones.

And then I'm guessing at some time later the using statement rule got changed and now applied, and maybe it got mis-merged into master-vs-deps again.

jasonmalinowski commented:
@JoeRobich's change to include this in the merge from master got in first, so I'll close this.
